### PR TITLE
Updating adviser funnel start page

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/cM2bchoj/3478-update-the-tta-content-to-cover-the-revamped-eea-service

### CODE REVIEW

**Comment by EllieNodder** [comment]
@gemmadallmandfe have made some changes based on feedback from Kat/Jade/Steph Roberts

**Comment by EllieNodder** [comment]
@gemmadallmandfe these changes still need to go live! 😬

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -4,33 +4,30 @@
...
+        <p>All our advisers have years of teaching experience and can help you whether you’re just thinking about teaching, or you’re ready to get back into the classroom.</p>
         <p>Your adviser can help you:</p>
         <ul>
           <li>get school experience</li>
-          <li>choose the right training</li>
+          <li>discover what teaching is really like</li>
           <li>write your personal statement</li>
           <li>find a teaching role</li>
           <li>find out how to improve your subject knowledge</li>
+          <li>understand your qualifications</li>
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/pages/home.html.erb:16]: Yep, think this was something the international team asked for. Have changed to 'understand the qualifications you need' - does this work better?

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -4,33 +4,30 @@
...
 
         <h2 class="govuk-heading-m">Non-UK trainees and teachers</h2>
 
         <p>To get an adviser, you'll need to have or be studying for an overseas qualification of the same standard as a UK bachelor’s degree, class 2:2 (honours) or higher.</p>
         
-        <p>You can check the UK equivalents of your degree qualifications by calling us on <%= link_to("0800 389 2500", "tel://08003892500") %>.</p>
+        <p>You can check the UK equivalents of your degree qualifications by calling us on <%= link_to("+44 800 389 2500", "tel://08003892500") %>.</p>
 
-        <p>If you're already qualified as a teacher outside the UK, you'll need to have <a href="https://www.gov.uk/guidance/qualified-teacher-status-qts">English qualified teacher status (QTS)</a> and want to teach a secondary subject to get an adviser.</p>
+        <p>If you're already qualified as a teacher outside the UK, you'll need to have <a href="https://www.gov.uk/guidance/qualified-teacher-status-qts">English qualified teacher status (QTS)</a> and want to teach a secondary subject (teaching children aged 11 to 16) to get an adviser.</p>
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/pages/home.html.erb:30]: If you're a returning teacher, you probably already know what QTS is, right? If so, we can remove the first link?

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -4,33 +4,30 @@
...
 
-        <p>To get an adviser, you'll need (one of the following):</p>
-
-        <ul>
-          <li>a bachelor's degree, class 2:2 (honours) or higher</li>
-          <li>to be studying for a degree and predicted to get a class 2:2 (honours) or higher</li>
-        </ul>
+        <p>You'll need to have or be studying for a bachelor's degree in any subject to get an adviser. If you already have a degree, it needs to be a 2:2 or higher.</p> 
         
         <p>If you’re thinking about returning to teaching, you'll need to have <a href="https://www.gov.uk/guidance/qualified-teacher-status-qts">qualified teacher status (QTS)</a> and want to teach a secondary subject in England to get an adviser.</p>
```
Comment (gemmadallmandfe) [inline app/views/pages/home.html.erb:22]:
```suggestion
        <p>If you’re thinking about returning to teaching, you'll need to have qualified teacher status (QTS) and want to teach a secondary subject in England to get an adviser.</p>
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**